regression/goto-analyzer-taint-ansi-c/taint-basic1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.o
+--taint taint.json
+^EXIT=0$
+^SIGNAL=0$
+^file main.c line 12( function .*)?: There is a T1 flow \(taint rule my_h1\)$
+^file main.c line 15( function .*)?: There is a T2 flow \(taint rule my_h2\)$
+--
```

Comment:
Maybe add checks that there is no T1 in line 15 and no T2 in line 12.

Reply:
Done.